### ISSUE TITLE
FEATURE - Activity/State Visibility - Change Batch

### ISSUE THREAD

**Issue by s38b35M5** [PrivacyDragon/Bookwyrm_Android#7]
Feature Request

Ability to select all or multiple books to change visibility/privacy of state among available states (Read, To Read, Reading, etc.)

This makes a change in desired privacy easier when many books have state visibility/privacy

**Comment by PrivacyDragon** [comment]
It is not completely clear to me what you mean.
Also, I am not sure whether it is possible to make it an option in the app itself.

**Comment by s38b35M5** [comment]
When I first joined bookwyrm, I set most of my book status to private, but now I'd like to change that to followers or public, which I seem to only be able to do by lengthy process with multiple clicks and unpredictable results. Sometimes, I can just click move to "read" and select a new visibility type and it saves. Other times I have to move the book to another shelf and back to change the visibility. 

If I could instead select all or various multiple books and change from private to another status, it would save a lot of time and make the platform feel more powerful.
